tests/js/spec/views/organizationIntegrations/integrationListDirectory.spec.jsx

```diff
@@ -0,0 +1,332 @@
+              doc: '',
+              featureDescriptions: [
+                {
+                  description: 'Forward Sentry errors and events to Amazon SQS.',
+                  featureGate: 'data-forwarding',
+                },
+              ],
+              features: ['data-forwarding'],
+              hasConfiguration: true,
+              id: 'amazon-sqs',
+              isHidden: false,
+              isTestable: false,
+              metadata: {},
+              name: 'Amazon SQS',
+              projectList: [],
+              resourceLinks: [
+                {
+                  title: 'Report Issue',
+                  url: 'https://github.com/getsentry/sentry/issues',
+                },
+                {
+                  title: 'View Source',
+                  url:
+                    'https://github.com/getsentry/sentry/tree/master/src/sentry_plugins',
+                },
+              ],
+              shortName: 'Amazon SQS',
+              slug: 'amazon-sqs',
+              status: 'beta',
+              type: 'data-forwarding',
+              version: '10.1.0.dev0',
+            },
+            {
+              status: 'unknown',
+              description: 'Send alerts to PagerDuty.',
+              isTestable: true,
+              isHidden: false,
+              hasConfiguration: true,
+              features: [],
+              shortName: 'PagerDuty',
+              id: 'pagerduty',
+              assets: [],
+              featureDescriptions: [],
+              name: 'PagerDuty',
+              author: {url: 'https://github.com/getsentry/sentry', name: 'Sentry Team'},
+              contexts: [],
+              doc: '',
+              resourceLinks: [
+                {
+                  url: 'https://github.com/getsentry/sentry/issues',
+                  title: 'Report Issue',
+                },
+                {
+                  url:
+                    'https://github.com/getsentry/sentry/tree/master/src/sentry_plugins',
+                  title: 'View Source',
+                },
+              ],
+              slug: 'pagerduty',
+              projectList: [
+                {
+                  projectId: 2,
+                  configured: true,
+                  enabled: true,
+                  projectSlug: 'javascript',
+                  projectPlatform: 'javascript',
+                  projectName: 'JavaScript',
+                },
+              ],
+              version: '10.1.0.dev0',
+              canDisable: true,
+              type: 'notification',
+              metadata: {},
+            },
+          ],
+        ],
+        [`/organizations/${org.slug}/repos/?status=unmigratable`, []],
+      ]);
+
+      wrapper = mountWithTheme(
+        <IntegrationListDirectory params={{orgId: org.slug}} location={{search: ''}} />,
+        routerContext
+      );
+    });
+    it('match previous snapshot', async function() {
+      expect(wrapper).toMatchSnapshot();
```

Comment:
I'm skeptical of how useful these snapshots are, imo it's more painful than useful, especially when you're snapshotting a large component with a 1k+ line snapshot file. Your test below tells me a lot more about what you deem important enough in the view that you're writing an explicit expectation for, whereas this snapshot can throw lots of false negatives (e.g. emotion class name changes, data changes).